docs/reference/components/sources/docker_logs.cue

```diff
@@ -84,6 +95,51 @@ components: sources: docker_logs: {
 	}
 
 	configuration: {
+		docker_host: {
+			common: true
+			description: """
+				The Docker host to connect to. Use an HTTPS URL to enable TLS encryption.
+				"""
+			required: false
+			type: string: {
+				default: null
+				examples: ["http://localhost:2375", "https://localhost:2376", "/var/run/docker.sock"]
+			}
+		}
+		tls: {
+			common:      false
+			description: "TLS options to connect to the Docker deamon. This has no effect unless `docker_host` is an HTTPS URL."
```

Comment:
Can we call out the env vars and path (`~/.docker`) that it will default to here?